config/src/cli.rs

```diff
@@ -171,6 +171,14 @@ pub struct Cli {
     /// The path at which to save raft snapshots
     #[clap(long, value_parser, default_value = "/raft_snapshots")]
     pub raft_snapshot_path: String,
+    /// Whether to record historical state locally
+    // TODO: Unset default `true` once event export implementation is complete
+    #[clap(long, value_parser, default_value = "true")]
+    pub record_historical_state: bool,
+    /// The address to export relayer events to, in multiaddr format.
+    /// If ommitted, the event manager is disabled.
+    #[clap(long, value_parser)]
+    pub event_export_addr: Option<String>,
```

Comment:
ah for credentials, that depends on the export method. for direct HTTP export yes, but for Unix socket, not necessarily. there doesn't seem to be a way for this to be folded into the `Multiaddr` so I'll add an optional config option for it

Reply:
as we discussed, it's a bit heavy-handed to enshrine auth patterns in the config. we'll leave the `event_export_addr` to be a `Multiaddr`, but it will be up to the relayer operator to manage their own authentication between the relayer and wherever they export events.